### DISCUSSION TITLE
Copy Prod DB data to Dev DB periodically

### DISCUSSION THREAD

**Discussion by bathientran** [supabase/supabase#2512] [Questions]
Hi everyone!

I'm currently trying to implement a dev version of the prod DB but want to have prod data in there to see how it interacts with the prod data. Is there a way to periodically copy the data from the DB and then rebuild the dev DB from that data?

I saw supabase's [publications](https://supabase.io/docs/reference/postgres/publications) but wanted to see if someone has a working pipeline/ flow that they could share?

Thanks a lot!

**Comment by silentworks** [comment]
You should always avoid copying production data to anywhere else, especially since sensitive information is likely to be in production. If you are talking about the database schema then there are tools to handle migrations.
